ansible/roles/test/files/ptftests/vnet_vxlan.py

```diff
@@ -192,14 +192,14 @@ def setUp(self):
         if 'routes_removed' in self.test_params and self.test_params['routes_removed']:
             self.routes_removed = True
 
-        if 'vxlan_srcport_range_enabled' in self.test_params and self.test_params['vxlan_srcport_range_enabled']:
-            self.vxlan_srcport_range_enabled = self.test_params['vxlan_srcport_range_enabled']
+        if 'vxlan_udp_sport' in self.test_params and self.test_params['vxlan_udp_sport']:
+            self.vxlan_srcport = self.test_params['vxlan_udp_sport']
```

Comment:
should we have this initialized?

Reply:
values saved in **self.test_params** are udp src port, mask and feature_enabled/disabled
if we will not set **self.vxlan_srcport** and other params there we will use their default values:
        self.vxlan_srcport= 0
        self.vxlan_srcport_mask = 8
        self.vxlan_srcport_range_enabled = False